lib/panel/controller.dart

```diff
@@ -181,11 +181,9 @@ class _SystemButton extends StatelessWidget {
   }
 }
 
-enum ButtonType { MoveLeft, RotateRight, MoveRight, Down, Drop }
-
 class _Button extends StatefulWidget {
   final Size size;
-  final ButtonType typeOfButton;
+  final IconData buttonIcon;
```

Comment:
the `buttonIcon` can be a `Widget` type. then the rotation problem could be easy to fix.
```dart
Button(
        enableLongPress: false,
        size: Size(90, 90),
        icon: Transform.rotate(angle: - (math.pi/ 12.0) * 3,child: Icon(Icons.file_download)),
        onTap: () {
          Game.of(context).drop();
        });
```